header/p2p/exchange_test.go

```diff
@@ -12,12 +11,11 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
-	tmbytes "github.com/tendermint/tendermint/libs/bytes"
-
 	"github.com/celestiaorg/go-libp2p-messenger/serde"
 
 	"github.com/celestiaorg/celestia-node/header"
 	p2p_pb "github.com/celestiaorg/celestia-node/header/p2p/pb"
+	headerMock "github.com/celestiaorg/celestia-node/nodebuilder/header/mocks"
```

Comment:
😠

Reply:
It's a custom mock, so we can keep it somewhere in to the header pkg

Reply:
Nodebuilder can be used to store autogen mocks

Reply:
thanks, fixed